provider/datagen/tests/data/postcard/fingerprints.csv

```diff
@@ -1050,6 +1050,24 @@ datetime/patterns/gregory/date@1, und-x-f, 26B, 1ef85443d9f4b70a
 datetime/patterns/gregory/date@1, und-x-l, 17B, 6a2ffebb5a23488e
 datetime/patterns/gregory/date@1, und-x-m, 17B, d389e08c6b881a75
 datetime/patterns/gregory/date@1, und-x-s, 17B, c3f8594394c6044b
+datetime/patterns/gregory/date_skeleton@1, ar, 737B, 70b716ffb4e168d9
```

Comment:
thought: this feels a bit big, but maybe that's fine.

Reply:
Should we split the 17 skeleta into smaller batches? They should deduplicate better...

Reply:
if we can come up with a good splitting scheme but I'm not set on it